web/modules/custom/dpl_loans/dpl_loans.info.yml

```diff
@@ -0,0 +1,10 @@
+name: DPL loans
+description: Handle the user profile loan pages
+package: DPL
+dependencies:
+  - drupal:block
+  - dpl_user_profile:dpl_user_profile
+  - dpl_fbs:dpl_fbs
+  - dpl_publizon:dpl_publizon
```

Comment:
Should this have a dependency to `dpl_react`?

Reply:
I dont know, should we?

Reply:
I've created a dependency on dpl_react, that must be the way to go